single_subject/batch_single_subject.sh

```diff
@@ -392,15 +396,62 @@ sct_warp_template -d fmri_moco_mean.nii.gz -w warp_template2fmri.nii.gz -a 0 -qc
 
 cd ../t1
 # Segment T1-weighted image (to be used in later steps)
-sct_deepseg_sc -i t1.nii.gz -c t1
+sct_deepseg -task seg_sc_contrast_agnostic -i t1.nii.gz
 
 # Smooth spinal cord along centerline (extracted from the segmentation)
 sct_smooth_spinalcord -i t1.nii.gz -s t1_seg.nii.gz
 # Tips: use flag "-sigma" to specify smoothing kernel size (in mm)
 
+# Second-pass segmentation using the smoothed anatomical image
+sct_deepseg_sc -i t1_smooth.nii.gz -c t1 -qc ~/qc_singleSubj
+
 # Align the spinal cord in the right-left direction using slice-wise translations.
 sct_flatten_sagittal -i t1.nii.gz -s t1_seg.nii.gz
 # Note: Use for visualization purposes only
 
+
+
+# New features (SCT v6.5, December 2024)
+# ======================================================================================================================
+
+# Lesion analysis for SCI and MS lesions
+cd ../t2_lesion
+# Segment the spinal cord and intramedullary lesion using the SCIsegV2 model
+# Note: t2.nii.gz contains a fake lesion for the purpose of this tutorial
+sct_deepseg -i t2.nii.gz -task seg_sc_lesion_t2w_sci -qc ~/qc_singleSubj
+# Note: Two files are output:
+# - t2_sc_seg.nii.gz: the spinal cord segmentation
+# - t2_lesion_seg.nii.gz: the lesion segmentation
+# Check results using FSLeyes
+fsleyes t2.nii.gz -cm greyscale t2_sc_seg.nii.gz -cm red -a 70.0 t2_lesion_seg.nii.gz -cm blue-lightblue -a 70.0 &
+# Compute various morphometric measures, such as number of lesions, lesion length, lesion volume, etc.
+sct_analyze_lesion -m t2_lesion_seg.nii.gz -s t2_sc_seg.nii.gz -qc ~/qc_singleSubj
+
+# Lesion analysis using PAM50 (the -f flag is used to specify the folder containing the atlas/template)
+# Note: You must go through the "Register to Template" steps (labeling, registration) first
+# sct_analyze_lesion -m t2_lesion_seg.nii.gz -s t2_sc_seg.nii.gz -f label -qc ~/qc_singleSubj
+
+# Note: We also have a contrast-agnostic segmentation command for MS lesions, too:
+sct_deepseg -i t2.nii.gz -task seg_ms_lesion -qc ~/qc_singleSubj
+# As well as a segmentation command tailored to MP2RAGE MS lesions
+sct_deepseg -i t2.nii.gz -task seg_ms_lesion_mp2rage -qc ~/qc_singleSubj
+
+# Rootlets segmentation
+cd ../t2
+# Segment the spinal nerve rootlets
+sct_deepseg -i t2.nii.gz -task seg_spinal_rootlets_t2w -qc ~/qc_singleSubj
+# Check results using FSLeyes
+fsleyes t2.nii.gz -cm greyscale t2_rootlets.nii.gz -cm subcortical -a 70.0 &
```

Comment:
idem -- after moving rootlets seg here, we need to remove the original command above:
https://github.com/spinalcordtoolbox/sct_tutorial_data/blob/3ecf3147cc7521389cb9b536ac79899470811c73/single_subject/batch_single_subject.sh#L136-L143